File: baekjoon/p04998/Main.java
package baekjoon.p04998;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while (sc.hasNext()) {
            System.out.println(solve(sc.nextDouble(), sc.nextDouble(), sc.nextDouble()));
        }
    }

    private static int solve(double n, double b, double m) {
        int ret = 0;
        while (n < m) {
            n += (n * b * 0.01);
            ret++;
        }

        return ret;
    }

}

File: baekjoon/p04998/Main.kt
package baekjoon.p04998

fun main() {
    while (true) {
        val input = readLine() ?: break
        val inputs = input.trim().split(" ").map { it.toDouble() }
        println(solve(inputs[0], inputs[1], inputs[2]))
    }
}

private fun solve(n: Double, b: Double, m: Double): Int {
    var ret = 0
    var nn = n

    while (nn < m) {
        nn += (nn * b * 0.01)
        ret++
    }

    return ret
}